docs/reference/ccr/getting-started.asciidoc

```diff
@@ -306,36 +324,34 @@ POST /server-metrics-follower/_ccr/unfollow
 [[ccr-getting-started-auto-follow]]
 ==== Automatically create follower indices
 Create <<ccr-auto-follow,auto-follow patterns>> to automatically follow time
-series indices that are periodically created in a remote cluster (such as daily
-{beats} indices).
+series indices that are periodically created in a remote cluster, such as daily
```

Comment:
This phrasing isn't quite right. It's not clear that they are automatically created via Rollover (or Data streams?). And I don't think we want to reinforce the notion that daily indices are a good pattern.